src/java.desktop/unix/native/libawt_xawt/awt/gtk2_interface.h

```diff
@@ -86,13 +86,6 @@ typedef struct {
   gushort   revents;
 } GPollFD;
 
-typedef struct {
-  gint x;
-  gint y;
-  gint width;
-  gint height;
-} GdkRectangle;
```

Comment:
Any reason why it is removed? I think it's needed in gtk2_interface.c, right? 
Probably we can delete from gtk3_interface.h as it is not used in gtk3_interface.c

Reply:
ok, I see it's used in screencast c file so moved to gtk_interface.h